it/utils/GuiceWiremockSuite.scala

```diff
@@ -30,12 +30,33 @@ import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.inject.guice.GuiceableModule
 import com.github.tomakehurst.wiremock.common.ConsoleNotifier
 
-trait WiremockSuite extends BeforeAndAfterAll with BeforeAndAfterEach with GuiceFakeApplicationFactory {
+trait WiremockSuite extends BeforeAndAfterAll with BeforeAndAfterEach {
```

Comment:
If we're using both WiremockSuite and GuiceWiremockSuite why is the file being renamed?

Reply:
Good spot. I used the IDE's rename function to rename the original WiremockSuite to rename the symbol everywhere it is used before splitting it, and I didn't realise it'd renamed the file too. I've undone the rename.